src/test/java/com/wikia/webdriver/common/core/ExtHelper.java

```diff
@@ -0,0 +1,64 @@
+package com.wikia.webdriver.common.core;
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.driverprovider.NewDriverProvider;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+
+import org.apache.commons.io.FileUtils;
+import org.openqa.selenium.WebDriverException;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collection;
+
+/*
+ Browser extensions helper
+ */
+public class ExtHelper {
+
+  private ExtHelper() {
+
+  }
+
+  public static void addExtension(String name) {
+    String br = Configuration.getBrowser();
```

Comment:
what about more meaningful variable name, instead of `br`, `browser` or `currentBrowser`?